docs/references/restatectl.mdx

```diff
@@ -198,3 +202,62 @@ Alive partition processors (nodes config v21, partition table v21)
  N3    11 minutes, 40 seconds and 995 ms ago
  ```
 </details>
+
+## Replicated Loglet
+
+Manage the replicated loglet using `restatectl replicated-loglet`. See [Restate Architecture](/references/architecture#distributed-log-aka-bifrost) for general background on Restate's distributed log.
+
+When you use the replicated loglet, which is required for distributed operation, the Restate control plane selects nodes on which to replicate the log according to the specified log replication. Each `log-server` node in the cluster has a storage state which determines how the control plane may use this node. The `set-storage-state` tool allows you to manually override this state as operational needs dictate.
+
+New log servers come up in the `provisioning` state and will automatically transition to `read-write`. The `read-write` state means that the node is considered both healthy to read from and accept writes, that is it may be selected as a node set member for new loglet segments.
+
+### list-servers
+
+You can view the current storage state of log servers in your cluster using the `list-servers` sub-command.
+
+```shell
+restatectl replicated-loglet list-servers
+```
+
+<details className="grey-details">
+    <summary>Output</summary>
+```
+Node configuration v12
+Log chain v3
+ NODE  GEN   STORAGE-STATE  HISTORICAL LOGLETS  ACTIVE LOGLETS
+ N1    N1:3  read-write     4                   2
+ N2    N2:2  read-write     4                   2
+ N3    N3:2  read-write     4                   2
+```
+</details>
+
+Other valid storage include `data-loss`, `read-only`, and `disabled`. Nodes may transition to `data-loss` if they detect that some previously written data is not available. This does not necessarily imply corruption, only that such nodes may not participate in some quorum checks. Such nodes may transition back to `read-write` if they can be repaired.
```

Comment:
Atm, we don't attempt to repair data loss. So `data-loss` is a terminal state. 

There is currently a problematic situation in which we transition the `StorageState` to `DataLoss` if the user accidentally starts a new node with an existing node name. In that case, the system thinks that this this node has lost its data. To recover from this situation, one needs to force set the `storage-state` to `ReadWrite`. That should be the only situation right now in which you need the `set-storage-state` command. Everything else (like draining, for example) does not really work yet because we need log chain trimming. Could you add the problematic scenario and how to get out of it? Maybe this also deserves a dedicated trouble shooting guide instead of the `restatectl` description?

Reply:
Follow up here: https://github.com/restatedev/documentation/pull/543